bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/CameraCapability.java

```diff
@@ -77,6 +82,35 @@ public void updateHomeStatusModule(HomeStatusModule newData) {
         }
     }
 
+    @Override
+    protected void updateWebhookEvent(WebhookEvent event) {
+        super.updateWebhookEvent(event);
+
+        final ThingUID thingUid = handler.getThing().getUID();
+        handler.updateState(new ChannelUID(thingUid, GROUP_SUB_EVENT, CHANNEL_EVENT_TYPE),
+                toStringType(event.getEventType()));
+        handler.updateState(new ChannelUID(thingUid, GROUP_SUB_EVENT, CHANNEL_EVENT_TIME),
+                toDateTimeType(event.getTime()));
+        handler.updateState(new ChannelUID(thingUid, GROUP_SUB_EVENT, CHANNEL_EVENT_SNAPSHOT),
+                toRawType(event.getSnapshotUrl()));
+        handler.updateState(new ChannelUID(thingUid, GROUP_SUB_EVENT, CHANNEL_EVENT_SNAPSHOT_URL),
+                toStringType(event.getSnapshotUrl()));
+        handler.updateState(new ChannelUID(thingUid, GROUP_SUB_EVENT, CHANNEL_EVENT_VIGNETTE),
+                toRawType(event.getVignetteUrl()));
+        handler.updateState(new ChannelUID(thingUid, GROUP_SUB_EVENT, CHANNEL_EVENT_VIGNETTE_URL),
+                toStringType(event.getVignetteUrl()));
+
+        final String message = event.getName();
+        handler.updateState(new ChannelUID(thingUid, GROUP_SUB_EVENT, CHANNEL_EVENT_MESSAGE),
+                message == null || message.isBlank() ? UnDefType.NULL : toStringType(message));
+
+        // The channel should get triggered at last (after super and sub methods), because this allows rules to access
+        // the new updated data from the other channels.
+        final String eventType = event.getEventType().name();
+        handler.getHomeHandler().ifPresent(homeHandler -> homeHandler.triggerChannel(CHANNEL_HOME_EVENT, eventType));
+        handler.triggerChannel(CHANNEL_HOME_EVENT, eventType);
```

Comment:
Does it mean we have a trigger channel on both things (home and camera).
There is a real need for that ?

Reply:
Yes, that is one of the purposes of the issue / pull-request. When someone has more than 1 camera, it isn't sufficient to have the channel on the home level, because nobody knows which camera has triggered that event. For example I want to get an alarm when someone is detected within the garden, when we are not at home. But I don't want to get an alarm when someone is at the door(bell). And it is also required to let a rule send the image of the right camera. I'm sending Telegram notifications with an image, therefore the image of the triggering camera should be send. ;-)

Reply:
My question was rather if it is necessary to keep the trigger event at home level?

Reply:
I think that could get refactored, but it would make the binding incompatible, there are more things which would need new trigger channels and maybe the global trigger channel is still useful when it isn't required to differ between the single devices. I wouldn't change that yet.